src/vim9.h

```diff
@@ -200,6 +200,7 @@ typedef enum {
     ISN_USEDICT,    // use or clear dict saved by ISN_MEMBER/ISN_STRINGMEMBER
 
     ISN_PUT,	    // ":put", uses isn_arg.put
+    ISN_IPUT,	    // ":iput", uses isn_arg.put
```

Comment:
Can you add a new test to the test_vim9_disassemble.vim file for this new instruction?